### PR TITLE
Add docs for Github exporter of issues, workflows and workflows runs

### PR DESCRIPTION
**Pull request by talsabagport**
# Description

Add docs for Github exporter of issues, workflows and workflows runs

## Added docs pages

Please also include the path for the added docs

- Github Exporter for Issues (`exporters/github-exporter/exporting-issues.md`)
- Github Exporter for Workflows (`exporters/github-exporter/exporting-workflows.md`)
- Github Exporter for Workflows Runs (`exporters/github-exporter/exporting-workflows-runs.md`)

## Updated docs pages

Please also include the path for the updated docs

- Github Exporter Self Hosted Installation (`exporters/github-exporter/self-hosted-installation.md`)

### CODE REVIEW

File: docs/exporters/github-exporter/github-exporter.md [hunk 1]
```diff
@@ -14,5 +14,8 @@ Our integration with GitHub allows you to export GitHub objects to Port as Entit
 
 ## Next Steps
 
-[Explore How to automatically import `pull-requests` from GitHub](./exporting-pull-requests.md)
-[Explore How to automatically import `repositories` from GitHub](./exporting-repositories.md)
+- [Explore How to automatically import `pull-requests` from GitHub](./exporting-pull-requests.md)
```
Comment (MPTG94) [inline docs/exporters/github-exporter/github-exporter.md:17]:
```suggestion
- [Automatically import `pull-requests` from GitHub](./exporting-pull-requests.md)
```

File: docs/exporters/github-exporter/github-exporter.md [hunk 2]
```diff
@@ -14,5 +14,8 @@ Our integration with GitHub allows you to export GitHub objects to Port as Entit
...
+- [Explore How to automatically import `repositories` from GitHub](./exporting-repositories.md)
```
Comment (MPTG94) [inline docs/exporters/github-exporter/github-exporter.md:18]:
```suggestion
- [Automatically import `repositories` from GitHub](./exporting-repositories.md)
```

File: docs/exporters/github-exporter/github-exporter.md [hunk 3]
```diff
@@ -14,5 +14,8 @@ Our integration with GitHub allows you to export GitHub objects to Port as Entit
...
+- [Explore How to automatically import `issues` from GitHub](./exporting-issues.md)
```
Comment (MPTG94) [inline docs/exporters/github-exporter/github-exporter.md:19]:
```suggestion
- [Automatically import `issues` from GitHub](./exporting-issues.md)
```

File: docs/exporters/github-exporter/github-exporter.md [hunk 4]
```diff
@@ -14,5 +14,8 @@ Our integration with GitHub allows you to export GitHub objects to Port as Entit
...
+- [Explore How to automatically import `workflows` from GitHub](./exporting-workflows.md)
```
Comment (MPTG94) [inline docs/exporters/github-exporter/github-exporter.md:20]:
```suggestion
- [Automatically import `workflows` from GitHub](./exporting-workflows.md)
```

File: docs/exporters/github-exporter/github-exporter.md [hunk 5]
```diff
@@ -14,5 +14,8 @@ Our integration with GitHub allows you to export GitHub objects to Port as Entit
...
+- [Explore How to automatically import `workflows-runs` from GitHub](./exporting-workflows-runs.md)
```
Comment (MPTG94) [inline docs/exporters/github-exporter/github-exporter.md:21]:
```suggestion
- [Automatically import `workflows-runs` from GitHub](./exporting-workflows-runs.md)
```

**Review by MPTG94 — APPROVED** [review]
LGTM, apply suggested fixes and ask for proof read

File: docs/exporters/github-exporter/exporting-workflows-runs.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+sidebar_position: 7
+title: Exporting workflows runs
+---
+
+:::note Prerequisites
+
+[Please install our GitHub app](./installation).
+:::
+
+In this tutorial, we will export workflows runs from GitHub and create matching Port Entities!
```
Comment (EllaFurman) [inline docs/exporters/github-exporter/exporting-workflows-runs.md:11]:
```suggestion
In this tutorial, we will export workflow runs from GitHub and create matching Port Entities!
```

**Review by EllaFurman — CHANGES_REQUESTED** [review]

**Pull request merged**